libraries/push/impl/src/main/AndroidManifest.xml

```diff
@@ -7,8 +7,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
-
     <uses-permission android:name="android.permission.REQUEST_IGNORE_BATTERY_OPTIMIZATIONS" />
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE_REMOTE_MESSAGING" />
```

Comment:
I hope it will not make the PlayStore to require more work like providing a video demoing the usage of this permission. It is a bit of a shame for supposedly dead code. Can't we just take the risk to remove our override of `getForegroundInfo()`? If we got crash report, we can put it back but it should not happen I believe.